tests/integration/targets/win_dns_zone/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Ensure AD/DNS roles are installed
+  ansible.windows.win_feature:
+    name:
+      - AD-Domain-Services
```

Comment:
Do we need to create a domain here, can DNS work just standalone? This is a very expensive thing to add to CI and I would lean to keeping the tests but just having it run manually by people.

Reply:
Unfortunately this is necessary in order to test AD integrated zones. I'm happy to separate the test so that it's not run in CI.

Reply:
Cool, to stop it running in CI just add the following to the new modules test `aliases` file. The commend part can be anything really just something that is descriptive of why we don't run it in CI.

```
unsupported  # Requires domain setup
```

It's great that you've got the tests though, if someone wants to validate it they can run it manually themselves, it's just a bit too long to run it in CI.